### PR TITLE
secpoll and changelog for auth 4.2.1

### PR DESCRIPTION
**Pull request by Habbie**
### Short description
-

### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/changelog/4.2.rst [hunk 1]
```diff
@@ -1,6 +1,94 @@
...
+    :tags: Performance, Improvements
+    :pullreq: 8457
+
+    API: reduce number of database connections (Kees Monshouwer)
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8546
+
+    register a few known RR types and remove an unknown one
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]:
```suggestion
    Register a few known RR types and remove an unknown one
```

File: docs/changelog/4.2.rst [hunk 2]
```diff
@@ -1,6 +1,94 @@
...
+    :pullreq: 8508
+    :tickets: 8218
+
+    pdnsutil increase-serial: under SOA-EDIT=INCEPTION-EPOCH, bump as if it is EPOCH
+
+  .. change::
+    :tags: Features, Improvements
+    :pullreq: 8547
+
+    add configurable timeout for inbound AXFR (Matti Hiljanen)
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]:
```suggestion
    Add configurable timeout for inbound AXFR (Matti Hiljanen)
```

File: docs/changelog/4.2.rst [hunk 3]
```diff
@@ -1,6 +1,94 @@
...
+    Basic validation of $GENERATE parameters
+
+  .. change::
+    :tags: Features, Improvements
+    :pullreq: 8428
+
+    Add CentOS 8 as builder target
+
+  .. change::
+    :tags: Features, Improvements
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]:
```suggestion
    :tags: New Features, Improvements
```

File: docs/changelog/4.2.rst [hunk 4]
```diff
@@ -1,6 +1,94 @@
...
+    API: optionally do not return dnssec info in domain list (Christian Hofstaedtler)
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8454
+
+    Basic validation of $GENERATE parameters
+
+  .. change::
+    :tags: Features, Improvements
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]:
```suggestion
    :tags: New Features, Improvements
```

File: docs/changelog/4.2.rst [hunk 5]
```diff
@@ -1,6 +1,94 @@
...
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8508
+    :tickets: 8218
+
+    pdnsutil increase-serial: under SOA-EDIT=INCEPTION-EPOCH, bump as if it is EPOCH
+
+  .. change::
+    :tags: Features, Improvements
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]:
```suggestion
    :tags: New Features, Improvements
```

File: docs/changelog/4.2.rst [hunk 6]
```diff
@@ -1,6 +1,94 @@
...
+    API: reduce number of database connections (Kees Monshouwer)
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8546
+
+    register a few known RR types and remove an unknown one
+
+  .. change::
+    :tags: Features, Improvements
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]:
```suggestion
    :tags: New Features, Improvements
```

File: docs/changelog/4.2.rst [hunk 7]
```diff
@@ -1,6 +1,94 @@
...
+    :pullreq: 8428
+
+    Add CentOS 8 as builder target
+
+  .. change::
+    :tags: Features, Improvements
+    :pullreq: 8341
+
+    gmysql backend, add an option to send the SSL capability flag
+
```
Comment (pieterlexis) [inline docs/changelog/4.2.rst]: Please add https://github.com/PowerDNS/pdns/pull/8576 after merging as well

**Comment by Habbie** [comment]
> Looks good, a few nits. The `Features` -> `New Features` changes can be avoided if we add `Features` to the tags in `docs/conf.py`, if I'm not mistaken.

Definitely, but I decided i like 'New Features' :)

**Comment by Habbie** [comment]
All comments addressed; rebased and squashed. Should be done now, except the date may shift to Thursday.

**Pull request merged**